.goreleaser.yml

```diff
@@ -38,8 +38,16 @@ brews:
     folder: Formula
     homepage:  https://roots.io/trellis
     description: A CLI to manage Trellis projects
+    custom_block: |
+      head "https://github.com/roots/trellis-cli.git"
+      depends_on "go" => :build
```

Comment:
As long as this means it's only needed for build/head 👍

Reply:
Good catch!
Seems a goreleaser tap always install build dependencyies.

Reply:
Seems goreleaser always install `depends_on "go" => :build`.

What do you think if we make another formula (trellis-cli-dev) in https://github.com/roots/homebrew-tap

Something like what sequel does:
```sh-session
$ brew search sequel
==> Casks
psequel                    sequel-pro                 sequel-pro-nightly ✔
```

Reply:
I'm good with that 👍 I think that just means adding another brew entry in the goreleaser config in this repo?

Reply:
I think https://github.com/roots/homebrew-tap/pull/1 is enough because that formula doesn't need to be changed by goreleaser